content/klc/F5.3.adoc

```diff
@@ -19,6 +19,7 @@ A fully enclosed Component courtyard must be drawn on the `F.CrtYd` layer, with
 . Courtyard uses `0.05mm` line width
 . All courtyard line elements are placed on a `0.01mm` grid
 . If the component requires a courtyard on the back of the PCB, a corresponding courtyard must be provided on the `B.CrtYd` layer.
+. The courtyard should be such that any permitted tolerances in the part component do not result in the part violating the clearance to the courtyard
```

Comment:
IPC defines that courtyard is relative to the nominal size of the part. (Meaning this is wrong)

Reply:
Changed.